### PR TITLE
use twig template for unauthorized acl messages

### LINKED ISSUE #5432: use twig template for unauthorized acl messages
use twig template for unauthorized acl messages

### PR DESCRIPTION
**Pull request by bradymiller**
fixes #5432

### CODE REVIEW

**Comment by stephenwaite** [comment]
lots of nice fixes!
bradys-panda

File: interface/drugs/add_edit_lot.php [hunk 1]
```diff
@@ -129,7 +130,8 @@ function genWarehouseList($tag_name, $currvalue, $title, $class = '')
   AclMain::aclCheckCore('inventory', 'consumption') ||
   AclMain::aclCheckCore('inventory', 'destruction');
 if (!$auth_lots) {
-    die(xlt('Not authorized'));
+    echo (new TwigContainer(null, $GLOBALS['kernel']))->getTwig()->render('core/unauthorized.html.twig', ['pageTitle' => xl("Edit/Add Lot")]);
+    exit;
```
Comment (bradymiller) [inline interface/drugs/add_edit_lot.php:134]: will try to move this to top of script

File: interface/orders/order_manifest.php [hunk 2]
```diff
@@ -349,7 +350,8 @@ function generate_order_summary($orderid)
 // Check authorization.
 $thisauth = AclMain::aclCheckCore('patients', 'med');
 if (!$thisauth) {
-    die(xlt('Not authorized'));
+    echo (new TwigContainer(null, $GLOBALS['kernel']))->getTwig()->render('core/unauthorized.html.twig', ['pageTitle' => xl("Order Summary")]);
+    exit;
```
Comment (bradymiller) [inline interface/orders/order_manifest.php:354]: will try to move this to top of script

File: interface/orders/pending_followup.php [hunk 3]
```diff
@@ -64,7 +65,8 @@ function thisLineItem($row, $codetype, $code)
 }
 
 if (! AclMain::aclCheckCore('acct', 'rep')) {
-    die(xlt("Unauthorized access."));
+    echo (new TwigContainer(null, $GLOBALS['kernel']))->getTwig()->render('core/unauthorized.html.twig', ['pageTitle' => xl("Pending Followup from Results")]);
+    exit;
```
Comment (bradymiller) [inline interface/orders/pending_followup.php:69]: will try to move this to top of script

**Pull request merged**